extended/src/test/java/apoc/load/LoadHtmlTestParameterized.java

```diff
@@ -78,13 +82,15 @@ public void testQueryH2WithConfig() {
         addBrowserIfSet(confList);
         Map<String, Object> config = map(confList.toArray());
 
-        testResult(db, "CALL apoc.load.html($url, $query, $config)",
-                map("url",new File("src/test/resources/wikipedia.html").toURI().toString(), "query", query, "config", config),
-                result -> {
-                    Map<String, Object> row = result.next();
-                    assertEquals(map("h2",asList(RESULT_QUERY_H2)).toString().trim(), row.get("value").toString().trim());
-                    assertFalse(result.hasNext());
-                });
+        skipIfBrowserNotPresentOrCompatible(() -> {
```

Comment:
What makes us require this? Local development?

Reply:
Yes exactly, for local testing